src/app/[locale]/(pages)/accedi/page.tsx

```diff
@@ -29,6 +28,41 @@ const Access = (): React.ReactElement => {
   const userFromToken = token && parseJwt(token) ? userFromJwtToken(token) : undefined;
   const loginInfo = storageLoginInfoOps.read();
 
+  // Function to replace the origin and redirect the user to the modified URL
+  const replaceOriginAndRedirect = useCallback(() => {
+    // Retrieve the current URL from the browser window
+    const currentUrl = window.location.href;
+
+    try {
+      // Create a URL object to safely manipulate the URL components
+      const url = new URL(currentUrl);
+
+      // Check if the current origin is "https://ioapp.it" or "http://localhost:3000"
+      switch (url.origin) {
+        case 'https://ioapp.it':
+          // If the origin is the production domain, replace it with "account.ioapp.it"
+          // eslint-disable-next-line functional/immutable-data
+          url.host = 'account.ioapp.it';
+          // Reassign the modified URL, which triggers a redirect to the new URL
+          window.location.assign(url.href);
+          break;
+        case 'http://localhost:3000':
+          // If the origin is localhost, replace it with "sub.localhost:3000" (testing scenario)
+          // eslint-disable-next-line functional/immutable-data
+          url.host = 'sub.localhost:3000';
```

Comment:
Why not using `account.localhost:3000`? I know it's just local development alias, but it'd be consistent with the production naming convention. What do you think?

Reply:
I had considered this option, but kept it generic voluntarily in order to have a ‘generic’ sub-domain configured locally and not necessarily linked to this project and this specific feature. What do you think?